cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsRdsModifyService.java

```diff
@@ -51,10 +48,38 @@ public void disableDeleteProtection(AuthenticatedContext ac, DatabaseStack dbSta
             throw new CloudConnectorException(ex.getMessage(), ex);
         }
 
+        waitUntilModifyFinishes(dbInstanceIdentifier, rdsClient, "Failed to disable deletion protection");
+        LOGGER.debug("RDS delete protection is disabled for DB Instance ID: {}", dbInstanceIdentifier);
+    }
+
+    public void updateMasterUserPassword(AuthenticatedContext ac, DatabaseStack databaseStack, String newPassword) {
+        AmazonRdsClient rdsClient = getAmazonRdsClient(ac);
+        String dbInstanceIdentifier = databaseStack.getDatabaseServer().getServerId();
+        ModifyDbInstanceRequest modifyDBInstanceRequest = ModifyDbInstanceRequest.builder()
+                .dbInstanceIdentifier(dbInstanceIdentifier)
+                .masterUserPassword(newPassword)
+                .build();
+        LOGGER.info("Modify master user password for database: {}", dbInstanceIdentifier);
```

Comment:
same question here regarding try block

Reply:
I just followed the already existing style and put only the provider calls in the try-catch block.

Reply:
discussed, makes sense